src/auth/SignUp.jsx

```diff
@@ -59,22 +60,24 @@ export const SignUp = () => {
     return undefined;
   };
 
-  const handleInputs = (e) =>
+  const handleInputs = (e) => {
     setNewUser({ ...newUser, [e.target.name]: e.target.value });
+    setErrorClient(null);
+    dispatchUser({ type: ERROR, payload: null });
+  };
 
   return (
     <SignUpStyles>
       <GridAuth height='800px'>
```

Comment:
I think we don' t do inline styles?

Reply:
You keep bringing this up but I am little confused with this comment to be honest.  Have you checked of how styled-components work [based](https://styled-components.com/docs/basics#adapting-based-on-props) on props? I always thought in order to use inline styles in React one should pass an style object to the component like so, `style={{...}}`.

Reply:
Did you check our `GridAuth`

```
export const GridAuth = styled.div`
  display: grid;
  grid-template-columns: 1fr 1fr;
  width: 1600px;
  height: ${(props) => props.height};
`;
```

Reply:
i was not aware of that

Reply:
> to use inline styles in React one should pass an style object to the component like so, style={{...}}

But that bit you knew it, right?

Reply:
yes